src/adera/Machines/Rocket.cpp

```diff
@@ -63,7 +63,6 @@ std::vector<WireOutput*> MachineRocket::existing_outputs()
 
 SysMachineRocket::SysMachineRocket(ActiveScene &scene) :
     SysMachine<SysMachineRocket, MachineRocket>(scene),
-    m_physics(scene.dynamic_system_find<SysPhysics>()),
     m_updatePhysics(scene.get_update_order(), "mach_rocket", "wire", "physics",
                     std::bind(&SysMachineRocket::update_physics, this))
```

Comment:
lambda?

Reply:
Hmm, I thought PR #35 would fix this, but I forgot.